src/mmw/js/src/modeling/gwlfe/entry/models.js

```diff
@@ -3,10 +3,16 @@
 var Backbone = require('../../../../shim/backbone'),
     GwlfeModificationModel = require('../../models').GwlfeModificationModel;
 
+var ENTRY_FIELD_TYPES = {
+    NUMERIC: 'NUMERIC',
+    YESNO: 'YESNO',
```

Comment:
This is probably fine but it seems like BOOLEAN would be a decent alternative name.

Reply:
Consider `Object.freeze`ing this enum.